GAME_DESIGN_PRINCIPLES.md

```diff
@@ -0,0 +1,100 @@
+# Cubyz is NOT a Survival Game!
+The goal of this project is to create a game about exploration and adventure, not survival. Our additions are inspired by reality, not copied from it.
+
+### Avoid Unavoidable Threats
+A player should have to create or approach a threat on their own terms, rather than the threat approach them.
+**This is a key part of Cubyz game design, as it is what keeps it from being a survival game.**
+
+**Examples:**
+- A player has lost much health, and must risk what little life they have left to gather food. Note that the player cannot heal unless they have an accessory that consumes their energy. This is a situation the player has to face because of their own skill level.
+- A player will not be attacked by monsters at night. They have to actively approach a monster-infested place such as a structure or cave. This is so they have free reign to do tasks during the night.
+- A player summons an event that spawns waves of enemies to their base.
+
+### Progression = Risk
+In order to progress in the game, the player must take risks. The player is otherwise free to stay where they are in terms of progression.
+
+**Examples:**
+- The deeper the player goes into the world's caves, the more valuables they can find, but consequentially, the more monsters they'll come across.
+- The player must defeat an enemy, boss, or event to obtain an item that helps them progress.
+- In a multiplayer world, a player might have to attack an enemy player to rise up in power.
+
+# Break the Cycle
+### No Dimensions
+Large dimension-like areas can be placed within the world itself, as it's immensely big already.
+"Dimensions" such as an underworld and an overworld are already planned for the game.
+
+### No Fast Travel
```

Comment:
I think this is worded poorly. There will be fast travel, just not fast travel in a twisted sense games often use that term - to avoid saying its teleportation. In Cubyz it will be fast air travel, fast railway travel etc. I would suggest `No Teleportation`